Cargo.toml

```diff
@@ -24,6 +24,7 @@ http2 = ["hyper/http2"]
 client = ["hyper/client"]
 tcp = ["hyper/tcp"]
 tls = ["native-tls", "openssl", "hyper-openssl", "hyper-tls"]
+uds = ["tokio", "tokio/net"]
```

Comment:
I _think_ you can just write `uds = ["tokio/net"]` (which implies `tokio`)

Reply:
I *think* I've seen that not work previously.

Reply:
Fine, it's not like this hurts, happy to leave as is